### PR TITLE
Backport fix for CVE-2022-41717 go1.17.13

### PR DESCRIPTION
**Pull request by rcrozean**
*Issue #, if available:*
#704 

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/golang/go/1.17/rpmbuild/SPECS/golang.spec [hunk 1]
```diff
@@ -157,6 +157,7 @@ Patch1:		0001-go-1.17.13-eks-archive-tar-limit-size-of-head.patch
 Patch2:		0002-go-1.17.13-eks-net-http-httputil-avoid-query-parameter-smuggling.patch
 Patch3:		0003-go-1.17.13-eks-regexp-limit-size-of-parsed-regexps.patch
 Patch4:     0004-go-1.17.13-eks-syscall-os-exec-reject-environ.patch
+Patch6:     0006-go-1.17.13-eks-net-http-update-bundled-golang.patch
```
Comment (kschumy) [inline projects/golang/go/1.17/rpmbuild/SPECS/golang.spec:160]: Why not 0005?

Comment (rcrozean) [reply projects/golang/go/1.17/rpmbuild/SPECS/golang.spec:160]: Knowing you were also doing patching.

**Pull request merged**